pkg/provision/options.go

```diff
@@ -111,6 +120,9 @@ type Options struct {
 	// Enable UEFI (for amd64), arm64 can only boot UEFI
 	UEFIEnabled bool
 
+	// Push the config using apply-config --insecure instead of serving it through the http server
+	ApplyConfig bool
```

Comment:
I don't think we need this to be part of the provision library, as this is done in `talosctl cluster create`?

Reply:
right, I forgot to remove it from this place